File: Resolvendo exercicíos em Python/secao03/Exercicio06.py
valor_por_hora = float(input("Qual valor você ganha por hora: "))
horas_trabalhadas = int(input("Quantas horas você trabalhou: "))

salario = horas_trabalhadas * valor_por_hora

print(f"Neste mês você vai receber R${salario:,.2f}")

File: Resolvendo exercicios em JAVA/src/secao03/Exercicio06.java
package secao03;

import java.util.Scanner;

public class Exercicio06 {

	public static void main(String[] args) {
		float valor_hora, salario;
		int horas_trabalhadas;
		Scanner teclado = new Scanner(System.in);
		
		System.out.println("Qual o valor você ganha por hora? ");
		valor_hora = teclado.nextFloat();
		
		System.out.println("Quantas horas você trabalhou neste mês? ");
		horas_trabalhadas = teclado.nextInt();
		
		salario = (horas_trabalhadas * valor_hora);
		
		System.out.println("Neste mês você vai receber R$:" + salario);
		
		teclado.close();
	}

}
